about.html

```diff
@@ -0,0 +1,124 @@
+                commons affilate network in a different country.This year,we hope to expand our invitation list including
+                organisations and individuals.<span>These speakers will be presenting on various
+                    aspects of Technology, from the initial imagination to a
+                    to bringing our imagination to life and solving problems.</span>
+            </p>
+            <h3>Please contact us via email for any further questions about <br>The coder CC Global summit 2023!</h3>
+            <a href="#">thecodersummit20@microverse.org</a>
+    </section>
+    <section id="cc-logo">
+        <div class="cc-container">
+            <h2>
+                The coder creative Common Global Summit 2023 logo</h2>
+            </h2>
+            <div class="TitleMark"></div>
+            <p>The logo of The coder summit 2023 was decided through the logo 
+                competition from 8. june to 7.july. <span>After online public voting and
+                    Selection Committee's judging process, the logo from Daniel Falcon won
+                    the prize. After consultation with Mr. Falcon, we optimized the
+                    prize-winning-logo for this year's summit as follows:</span>
+            </p>
+            <img src="./images/logo.jpg" alt="cc logo">
+        </div>
+    </section>
+     <!-- -------------------- Summits -------------------- -->
+    <section id="summit">
+        <div class="summit-container">
+            <h2>
+                See Previous Summit
+            </h2>
+            <div class="TitleMark"></div>
+            <p>
+                Take a look at the past two Coder Seminar which took
+                place in 2018 & 2021.
+            </p>
+            <div class="summit-card">
+                <div class="img">
+                    <h2>2018</h2>
+                    <p>The codee summit in Paris</p>
+                 </div>
+                 <div class="img">
+                    <h2>2021</h2>
+                    <p>The codee summit in Boston</p>
+                 </div>
+            </div>
+        </div>
+    </section>
+        <!-- -------------------- Partner -------------------- -->
+    <section id="partners">
+        <div class="title-container">
+            <h2 class="title">Partner</h2>
+            <div class="TitleMark"></div>
+            <div class="items">
+                <ul class="item-list">
+                    <li><img src="./images/mozilla 1.png" alt="partners"></li>
+                    <li><img src="./images/googlekorea 1.png" alt="partners"></li>
+                    <li><img src="./images/naver 1.png" alt="partners"></li>
+                    <li><img src="./images/daumkakaologo 1.png" alt="partners"></li>
+                    <li><img src="./images/airbnb 1.png" alt="partners"></li>
+                </ul>
```

Comment:
- [ ] Great job working so far but I think in the about page in the mobile version you forget to align the items as row you showed it as column but it should be as a row you can use the propert flex wrap to make it the same as the given design and flex-direction: row; and to align the items to be centered so please fix it to match the given design 🤩💯

- [ ] I think there is no red line after each icon from the partner icons I could see you added a border bottom but I think you can remove it since there is not red line in the given design and we should stick with given design as much as possible, shouldn't we? 💯🤩🔥💪🏻🙏🏻


- Your
![image](https://github.com/Shabiblac/CAPSTONE-1/assets/58553711/8a62403e-569e-47c0-80cb-e236697ff7b2)


- Given
![image](https://github.com/Shabiblac/CAPSTONE-1/assets/58553711/dcbe0726-2ec1-4ae0-9521-29be59fb38db)